wordcount_example.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+python -m apache_beam.examples.wordcount_minimal --input words.txt --output wordcounts.txt
```

Comment:
Is this actually running _our_ example code or some example pipeline built in to beam? I'd expect we'd need to reference `wordcount_example.py` to run it here.

As far as I can tell, the tutorial _does not include_ instructions for running your own Python driver file. What??

Reply:
#2 addresses this. The original intent of the shell file was simply to run the example provided by beam. This was primarily to ensure that everything was installed correctly.